### PR TITLE
Enabled visual style and fixed hidpi for windows

### PR DESCRIPTION
**Pull request by Srinivasa314**
Fixed tinyfiledialogs being ugly/blurry in windows

### CODE REVIEW

File: libtinyfiledialogs/tinyfiledialogs.c [hunk 1]
```diff
@@ -209,6 +209,14 @@ char const tinyfd_needs[] = "\
 #pragma warning(disable:4706) /* allows usage of strncpy, strcpy, strcat, sprintf, fopen */
 #endif
 
+#ifdef _WIN64
+#include <ShellScalingApi.h>
+
+#pragma comment(linker,"\"/manifestdependency:type='win32' \
```
Comment (nabijaczleweli) [inline libtinyfiledialogs/tinyfiledialogs.c:215]: Have you verified this against a non-MSVC toolchain?

Comment (Srinivasa314) [reply libtinyfiledialogs/tinyfiledialogs.c:215]: No

**Comment by jdm** [comment]
Have you submitted this patch upstream?

**Comment by Srinivasa314** [comment]
No,I haven't

**Comment by jdm** [comment]
In that case, we'll need to add these changes to a patch file in patches/ as well, so that they aren't overridden the next time we pull from upstream using update.sh.

**Comment by Srinivasa314** [comment]
Ok

**Comment by Srinivasa314** [comment]
The reason the build failed is because static.rust-lang.org was probably temporarily down.

**Comment by jdm** [comment]
Thanks!

**Pull request merged**